iridauploader/api/api_calls.py

```diff
@@ -351,6 +255,9 @@ def _get_irida_exception(response):
                                                 "{status_code}: {err_msg}\n".format(
                                                     status_code=str(response.status_code),
                                                     err_msg=response.reason))
+        elif response.status_code == HTTPStatus.NOT_FOUND:
+            e = exceptions.IridaResourceError("The url you specified could not be reached. Please verify your URL and "
+                                              "the projects/samples/ect you specified.")
```

Comment:
Very minor but should it be `etc` instead of `ect`, I am assuming that you mean etcetera.

Reply:
Good catch. Fixed.